configuration/configuration_blackbox_test.go

```diff
@@ -89,7 +89,7 @@ func TestGetKeycloakEndpointSetByUrlEnvVaribaleOK(t *testing.T) {
 func TestGetKeycloakEndpointAdminDevModeOK(t *testing.T) {
 	resource.Require(t, resource.UnitTest)
 	t.Parallel()
-	checkGetKeycloakEndpointOK(t, "http://sso.demo.almighty.io/auth/admin/realms/fabric8", config.GetKeycloakEndpointAdmin)
+	checkGetKeycloakEndpointOK(t, "http://sso.prod-preview.openshift.io/auth/admin/realms/fabric8", config.GetKeycloakEndpointAdmin)
```

Comment:
This problem with no existing request is not related to what I meant here.
The problem with missing request prevents us from starting obtaining public key from. It's a problem for both modes. Dev and production.
What I meant here is that in the PR I mentioned I exposed the dev mode sso URL via an exported method so we can re-use it directly in the test instead of hardcoding it everywhere.